torch/jit/annotations.py

```diff
@@ -291,7 +291,8 @@ def try_ann_to_type(ann, loc):
     if is_dict(ann):
         key = try_ann_to_type(ann.__args__[0], loc)
         value = try_ann_to_type(ann.__args__[1], loc)
-        return DictType(key, value)
+        if key and value:
```

Comment:
What do you think about generate error message here? IMHO, "unknown type name" might not be clear enough to users since Dict is a valid type.

Reply:
I added warnings let me know what you think.